addons/mail/static/src/bugfix/bugfix.js

```diff
@@ -7,4 +7,19 @@
 odoo.define('mail/static/src/bugfix/bugfix.js', function (require) {
 'use strict';
 
+const { CrashManager } = require('web.CrashManager');
+const { _lt } = require('web.core');
+
+CrashManager.include({
+    /**
+     * @override
+     */
+    init() {
+        this._super.apply(this, arguments);
+        this.odooExceptionTitleMap = Object.assign({}, this.odooExceptionTitleMap, {
+            'odoo.addons.base.models.ir_mail_server.MailDeliveryException': _lt("MailDeliveryException")
+        });
```

Comment:
That looks like a complex way to just write this
```suggestion
        Object.assign(this.odooExceptionTitleMap, {
            'odoo.addons.base.models.ir_mail_server.MailDeliveryException': _lt("MailDeliveryException")
        });
```